dcgm/README.md

```diff
@@ -66,21 +127,130 @@ sudo docker run --pid=host --privileged -e DCGM_EXPORTER_INTERVAL=3 --gpus all -
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: dcgm-exporter-read-datadog-cm
+  namespace: default
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  resourceNames: ["datadog-dcgm-exporter-configmap"]
+  verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: dcgm-exporter-datadog
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: dcgm-datadog-dcgm-exporter
+  namespace: default
+roleRef:
+  kind: Role 
+  name: dcgm-exporter-read-datadog-cm
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: datadog-dcgm-exporter-configmap
+  namespace: default
+data:
+  metrics: |
+      # Copy the content from the Installation section.
+```
+3. Create your DCGM Exporter Helm chart `dcgm-values.yaml` with the following content : 
+```yaml
+# Exposing more metrics than the default for additional monitoring - this requires the use of a dedicated ConfigMap for which the Kubernetes ServiceAccount used by the exporter has access thanks to step 1.
+# Ref: https://github.com/NVIDIA/dcgm-exporter/blob/e55ec750def325f9f1fdbd0a6f98c932672002e4/deployment/values.yaml#L38
+arguments: ["-m", "default:datadog-dcgm-exporter-configmap"]
+
+# Datadog Autodiscovery V2 annotations
+podAnnotations:
+  ad.datadoghq.com/exporter.checks: |-
+    {
+      "dcgm": {
+        "instances": [
+          {
+            "openmetrics_endpoint": "http://%%host%%:9400/metrics"
+          }
+        ]
+      }
+    }
 
-- https://github.com/iliakur/dcgm-exporter#quickstart-on-kubernetes
+# Optional - Disabling the ServiceMonitor which requires Prometheus CRD - can be re-enabled if Prometheus CRDs are installed in your cluster
+serviceMonitor:
+  enabled: false
+```
+4. Install the DCGM Exporter Helm chart in the `default` namespace with the following command, while being in the directory with your `dcgm-values.yaml` :
+```bash
+helm install dcgm-datadog gpu-helm-charts/dcgm-exporter -n default -f dcgm-values.yaml
+```
 
+**Note**: You can modify the release name `dcgm-datadog` as well as the namespace, but you must modify accordingly the manifest from step 1.
 
 <!-- xxz tab xxx -->
 <!-- xxx tab "Operator" xxx -->
 
-#### Operator
+#### Kubernetes (NVIDIA GPU Operator)
 
-To configure the Exporter in an Operator environment, please review the template provided by NVIDIA here:
+The DCGM exporter can be installed in a Kubernetes environment by using NVIDIA GPU Operator. The instructions below are derived from the template provided by NVIDIA [here](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html).
 
-- https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/getting-started.html#gpu-telemetry
+1. Add the NVIDIA GPU Operator Helm repository and ensure it is up-to-date :
+```bash
+helm repo add nvidia https://helm.ngc.nvidia.com/nvidia && helm repo update
+```
```

Comment:
```suggestion
   ```
```

Reply:
@rtrieu thanks for all the feedback/changes, just not understanding this one, we want to close this block of code ?

Reply:
hi @tbavelier sorry I don't know why it's suggesting to remove it - what I meant to say was to indent it by 3 lines to line it up with the numbered bullet, not to delete it :)

Reply:
Ah, that makes more sense, thanks a lot! Dismissed your review as I aligned everything now to be inside the number bullet, that's cleaner!